osu.iOS/Info.plist

```diff
@@ -34,9 +34,9 @@
 	<key>CADisableMinimumFrameDurationOnPhone</key>
 	<true/>
 	<key>NSCameraUsageDescription</key>
-	<string>We don&apos;t really use the camera.</string>
+	<string>We don't really use the camera.</string>
```

Comment:
I used rider's plist editor to make these changes and assume it knows what it's doing.